js/app/improve-this-page.js

```diff
@@ -134,34 +85,35 @@ $(document).ready(function () {
 
         var postData = $("#feedback-form-container").serialize();
         var postObject = new Object();
-        postObject.is_page_useful = false;
+        postObject.is_page_useful = true;
```

Comment:
Why are we switching the default to true?

Reply:
Since this is only sending feedback from the footer, it is not a general feedback, which would set the value for `Feedback Type' either `The whole website` or `A specific page`

switching to true tells `feedback-api` which service is attempting to send feedback and it uses the appropriate string for `Feedback Type`

Reply:
I'm not sure I follow that, maybe we should run through it.

Reply:
As discussed - revert this back to false.